stl/error_handler.py

```diff
@@ -0,0 +1,73 @@
+import collections
+
+
+ErrorInfo = collections.namedtuple('ErrorInfo', ['id', 'filename', 'line', 'position', 'message'])
+# Args:
+#   id: An identifier for the type of error.
+#   filename: Name of the file the error occured in.
+#   line: The text of the line the error occured in.
+#   position: An ErrorPosition pointing to where the error occured.
+#   messasge: The human readable message explaining the error
```

Comment:
I made it up. I wanted to have some documentation for the different fields, but I haven't seen docstrings for namedtuples, so I used a similar format of the 'Args' section from the normal docstring. Open to other formats